OpenRA.Mods.Common/Traits/Sellable.cs

```diff
@@ -53,9 +53,9 @@ public void Sell(Actor self)
 
 			var makeAnimation = self.TraitOrDefault<WithMakeAnimation>();
 			if (makeAnimation != null)
-				makeAnimation.Reverse(self, new Sell());
+				makeAnimation.Reverse(self, new Sell(), true);
 			else
-				self.QueueActivity(new Sell());
+				self.QueueActivity(true, new Sell());
```

Comment:
Same here as in the comment above, really: `QueueActivity(true, new SomeActivity())` is the same as just `QueueActivity(new SomeActivity())`. It only behaves differently if the `queued` paramter is false.

So I don't quite get what you want to achieve with this change. The behaviour with this patch is exactly the same as without it.